custom/icds_reports/templates/icds_reports/icds_app/mobile/sortable-kpi.directive.html

```diff
@@ -0,0 +1,26 @@
+{% load staticfiles %}
+<div ng-repeat="section in $ctrl.data">
+  <div class="shadow rounded-border bg-white col-xs-11 mx-auto float-unset mb-3 px-0 pb-3">
+    <div ng-click="$ctrl.toggleCardExpansion($index)">
+      <div class="d-flex justify-content-between align-items-center p-3" ng-click="$ctrl.expandCard($index)">
```

Comment:
where is `expandCard` defined?

Reply:
I have changed the function name to `toggleCardExpansion` and moved it to the `div` above it. but didnot remove it.